data/nmstate/002-nmstated.yaml

```diff
@@ -0,0 +1,46 @@
+---
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: nm-state-controller
+  namespace: nmstate-handler
+spec:
+  template:
+    metadata:
+      labels:
+        nmstate.io: state-controller
+    spec:
+{{ if .EnableSCC }}
+      serviceAccountName: nmstate-state-controller
```

Comment:
please change the service account to `nmstate-controller`

Reply:
per discussion above, keeping this the same